modules/client-handler/src/main/java/org/apache/ignite/client/handler/ClientHandlerMetricSource.java

```diff
@@ -0,0 +1,399 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.client.handler;
+
+import java.util.Arrays;
+import java.util.List;
+import org.apache.ignite.internal.metrics.AbstractMetricSource;
+import org.apache.ignite.internal.metrics.AtomicLongMetric;
+import org.apache.ignite.internal.metrics.Metric;
+import org.apache.ignite.internal.metrics.MetricSetBuilder;
+
+/**
+ * Server-side client handler metrics.
+ */
+public class ClientHandlerMetricSource extends AbstractMetricSource<ClientHandlerMetricSource.Holder> {
+    /**
+     * Constructor.
+     */
+    public ClientHandlerMetricSource() {
+        super("client-handler");
```

Comment:
It is not stated explicitly but I think that usage of dash (`-`) in a source or a metric name is not good idea because some 3rd-party tools can treat dash as some operator. But this is the recommendation, not a requirement

Reply:
Renamed to `clienthandler`. Or should it be `client.handler`?